nasa-gibs/worldview
# WV-3033 Dynamic Surface Water Extent story bug

## Description

[OPERA Surface Water Extent Story](https://worldview.uat.earthdata.nasa.gov/?v=-174.25840338686484,-95.92187266195343,141.53238355885279,97.3266290686873&l=Coastlines_15m,Reference_Features_15m,Reference_Labels_15m,OPERA_L3_Dynamic_Surface_Water_Extent-HLS_Provisional(disabled=5),HLS_L30_Nadir_BRDF_Adjusted_Reflectance(hidden),HLS_S30_Nadir_BRDF_Adjusted_Reflectance(hidden),Land_Water_Map&lg=true&tr=surface_water_extent&t=2023-09-16-T19%3A28%3A37Z), Step 4 displays white on the left side of the map, when it should be off as it has been set in the layer. 

Steps to recreate:

Load Story [surface water extent](https://worldview.uat.earthdata.nasa.gov/?v=-174.25840338686484,-95.92187266195343,141.53238355885279,97.3266290686873&l=Coastlines_15m,Reference_Features_15m,Reference_Labels_15m,OPERA_L3_Dynamic_Surface_Water_Extent-HLS_Provisional(disabled=5),HLS_L30_Nadir_BRDF_Adjusted_Reflectance(hidden),HLS_S30_Nadir_BRDF_Adjusted_Reflectance(hidden),Land_Water_Map&lg=true&tr=surface_water_extent&t=2023-09-16-T19%3A28%3A37Z)
    Go to Step 4
    See that the left "A" side is showing white (not water class) on the map, when it should actually be turned off.

## How To Test

- `git checkout WV-3033-dswe-story-bug`
- `npm run build`
- `npm run watch`
- go to this [link](http://localhost:3000/?v=-158.04624921237877,-78.60144071577226,108.17754621356465,69.14660315776926&l=Reference_Labels_15m,Coastlines_15m,OPERA_L3_Dynamic_Surface_Water_Extent-HLS_Provisional(disabled=5-3-0-4),HLS_L30_Nadir_BRDF_Adjusted_Reflectance(hidden),HLS_S30_Nadir_BRDF_Adjusted_Reflectance&lg=true&l1=Reference_Labels_15m,Coastlines_15m,OPERA_L3_Dynamic_Surface_Water_Extent-HLS_Provisional(disabled=5-0-3-4),HLS_L30_Nadir_BRDF_Adjusted_Reflectance(hidden),HLS_S30_Nadir_BRDF_Adjusted_Reflectance&lg1=true&ca=false&tr=surface_water_extent&t=2023-09-15-T02%3A00%3A00Z&t1=2023-12-19-T19%3A06%3A41Z)
- navigate to step 4 and confirm that the white band does not show up
- confirm that the rest of the story behaves normally

## PR Submission Checklist

This is simply a reminder of what we are going to look for before merging your code.

- I have read the [CONTRIBUTING](https://github.com/nasa-gibs/worldview/blob/main/.github/CONTRIBUTING.md) doc
- I have added necessary documentation (if applicable)
- I have added tests that prove my fix is effective or that my feature works (if applicable)
- Any dependent changes have been merged and published in downstream modules (if applicable)

## Merging

Please use the `squash and merge` commit method unless each commit in your branch is vital to the commit history of main.

@nasa-gibs/worldview
